samples/meetings-token-app/csharp/Service/MeetingService.cs

```diff
@@ -33,18 +34,18 @@ public class MeetingService : IMeetingService
         /// </summary>
         /// <param name="httpClientFactory">The Http client factory.</param>
         /// <param name="botCredentials">The Bot credentials.</param>
-        /// <param name="botAdapter">The Bot Framework HTTP adapter.</param>
+        /// <param name="adapters">The Bot Framework HTTP adapter.</param>
         /// <param name="tenantInfoRepository">The TenantInfo repository.</param>
         /// <param name="meetingServiceOptions">App options related to the meetings service.</param>
         public MeetingService(
             IHttpClientFactory httpClientFactory,
             AppCredentials botCredentials,
-            IBotFrameworkHttpAdapter botAdapter,
+            IBotFrameworkHttpAdapter adapters,
```

Comment:
It's a single adapter, please rename it like `botAdapter` or `botFrameworkHttpAdapter`

Reply:
Fixed